backend2/src/db/models/article.py

```diff
@@ -61,6 +61,7 @@ class ArticleData(Base, Timestamp):
     url = Column(UnicodeText, nullable=False)
     abstract = Column(UnicodeText, nullable=False, server_default='')
     mutations = relationship("ArticleMutation", back_populates="article_data")
+    publish_date = Column(Date, nullable=False)
```

Comment:
For the sake of migration this field should have a default value. `null` sounds good enough for me.

Reply:
By the way, I can't find the migration file in the PR.